srcpkgs/cegui/template

```diff
@@ -8,7 +8,7 @@ makedepends="libxml2-devel boost-devel glu-devel glew-devel glfw-devel
  devil-devel ois-devel silly-devel fribidi-devel lua51-devel minizip-devel
  $(vopt_if irrlicht irrlicht-devel)
  $(vopt_if ogre libogre-devel)
- $(vopt_if python 'python-devel boost-python')
+ $(vopt_if python 'python-devel boost-python1.69')
```

Comment:
boost-python1.69 is pulled in by `boost-devel`, so `python-devel` should be enough.

Reply:
Yeah, true, good point.